### DISCUSSION TITLE
topics with many subscribers

### DISCUSSION THREAD

**Discussion by wannastar** [eProsima/Fast-DDS#3940] [Q&A]
For typical use-case topics with many subscribers, as official documents, we should only setup DataReaderQos to add a multicast_locator. i,m wonderring whether we should to setup datawriter and how to choose multicast locator's IP and port?

**Comment by JesusPoderoso** [comment]
Hi @wannastar, the discovery locators are configured in the participant QoS. The participant that owns the endpoint (DataWriter or DataReader) can change its multicast locator by changing the builtin metatraffic multicast locators ([example here](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/listening_locators.html#metatraffic-multicast-locators)).

**Comment by wannastar** [comment]
thanks for your reply. What I'm wonderring is whether I should setup the datawriter in the below scenario
[https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/many_subscribers/many_subscribers.html#use-case-manysubscribers]

**Comment by JesusPoderoso** [comment]
The locator configures where the endpoint is listening traffic (discovery traffic or data communication traffic). Just by configuring the DataReaders locators with the custom multicast IP and port would make DataWriter communicate with them through that configuration. So, there is nothing to configure on the DataWriter side

**Comment by wannastar** [comment]
okay, thank you very much.
